jest.config.js

```diff
@@ -1,7 +1,18 @@
 /** @type {import('ts-jest').JestConfigWithTsJest} */
 
+const modules = ['@mui', '@babel'].join('|')
+
 module.exports = {
   preset: 'ts-jest',
-  testEnvironment: 'node',
+  testEnvironment: 'jsdom',
```

Comment:
have you checked out [happy dom](https://github.com/capricorn86/happy-dom)? i tried it out recently and had a good experience. i believe it's considered a bit faster, though i don't know _all_ the pros & cons to be honest. just thought i'd throw it out there though